spec/lib/traject/config/item_info_spec.rb

```diff
@@ -837,66 +782,6 @@
           expect(result['item_display_struct'].map { |x| JSON.parse(x) }.first['note']).to eq '.PuBlIc. Note'
         end
       end
-
-      context 'when the public note does not have periods around it' do
```

Comment:
These are currently not doing anything as the helper method `marc_to_folio_with_stubbed_holdings` is removing these before they become part of the FolioRecord.